.github/workflows/run-tests.yml

```diff
@@ -20,10 +20,24 @@ on:
 
 jobs:
   run-tests:
-    name: Run tests job
+    name: Run tests job matrix
 
     runs-on: ubuntu-latest
```

Comment:
thought: We could also include this in the matrix at somepoint if we want (I would like this).